inference-engine/tests_deprecated/unit/engines/mkldnn/graph/layers/internal/graph_split_test.cpp

```diff
@@ -230,27 +230,27 @@ INSTANTIATE_TEST_CASE_P(
                 split_test_params {
                         {1, 24, 2, 5},
                         {{1, 16, 2, 5}, {1, 8, 2, 5}},
-                        1, 5, MKLDNNPlugin::impl_desc_type::ref, {MKLDNNPlugin::impl_desc_type::ref}
```

Comment:
Comment for whole file:
// Copyright (C) 2018-2021 Intel Corporation

Reply:
Fixed. Thanks!